res/css/views/elements/_Pill.scss

```
@@ -47,6 +47,7 @@ limitations under the License.
     }
 
     .mx_BaseAvatar {
+        margin-left: -0.3em; // Otherwise the gap is too small
```

Did you mean "too large"?

---

```suggestion
        margin-left: -0.3em; // Otherwise the gap is too large
```

Ah, yep :man_facepalming: